twemproxy/tests/test_twemproxy.py

```diff
@@ -9,9 +10,28 @@
 SC_TAGS = ['host:{}'.format(common.HOST), 'port:{}'.format(common.PORT), 'optional:tag1']
 
 
-def test_check(check, dd_environment, setup_request, aggregator):
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_integration(check, dd_environment, setup_request, aggregator):
     check.check(common.INSTANCE)
+    _test_check(aggregator)
 
+    for stat in metrics.INTEGRATION_SERVER_STATS:
+        aggregator.assert_metric("twemproxy.{}".format(stat), count=2)
+    for stat in metrics.INTEGRATION_POOL_STATS:
+        aggregator.assert_metric("twemproxy.{}".format(stat), count=1)
+    aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check, setup_request, instance):
+    aggregator = dd_agent_check(instance)
```

Comment:
You should pass `rate` here. The reason you get less metrics is that those are rates.

Reply:
But the integration test is calling check just once :/

Reply:
Just confirming, adding `rate=True` breaks it in a different way

Reply:
Yes, it breaks in a different way because the `count` is going to be different.

Reply:
Yes, what I'm saying is that the counts are still different. If I set rate in e2e those metrics appear but with the same count as if there is no rate in integration. It is still better to have the rate and have them checked though